CMakeLists.txt

```diff
@@ -12,6 +12,9 @@ if (POLICY CMP0054)
   # Only interpret `if` arguments as variables or keywords when unquoted.
   cmake_policy(SET CMP0054 NEW)
 endif ()
+if( POLICY CMP0053 )
+    cmake_policy(SET CMP0053 NEW)
+endif( )
```

Comment:
nit: here and below, please follow the coding style of this `CMakeLists.txt`, in particular avoid extra spaces within parentheses and use 2-space indent:

```cmake
if (POLICY CMP0053)
  cmake_policy(SET CMP0053 NEW)
endif ()
```